saleor/graphql/warehouse/types.py

```diff
@@ -114,6 +114,10 @@ class Warehouse(ModelObjectType[models.Warehouse]):
         required=True,
         description="Shipping zones supported by the warehouse.",
     )
+    stocks = FilterConnectionField(
+        "saleor.graphql.warehouse.types.StockCountableConnection",
+        description="Stocks belonging to this warehouse.",
+    )
```

Comment:
@IKarbowiak We also need to figure out permission for this field, since `stock` / `stocks` queries have permissions, so we should check how warehouses can be queried and what permissions should be added here.